app/controllers/verify/sessions_controller.rb

```diff
@@ -31,6 +45,36 @@ def submit_profile
       end
     end
 
+    def confirm_profile_with_vendor
+      resolution = start_idv_session
+      track_idv_event(resolution)
+      process_resolution(resolution)
+    end
+
+    def process_resolution(resolution)
+      if resolution.success
+        idv_session.resolution = resolution
+        redirect_to verify_finance_path
+      elsif idv_attempter.exceeded?
+        redirect_to verify_fail_path
+      else
+        @failed_resolution = true
+        render :new
```

Comment:
It will likely be factored away, yes. Retries will happen at each step, since upstream confirmation now happens at each step. There is no "global retry" page.